### DISCUSSION TITLE
Cannot read properties of undefined import bulk [Jun 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2508] [Discord Threads]
> 🙋‍♂️ **fredkgb**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### please create an issue on github and provide a sample csv  
#  
> 🙋‍♂️ **wkw**  
#### are you importing data to an existing table?  
#  
> 🙋‍♂️ **fredkgb**  
#### Yes  
#  
> 🙋‍♂️ **fredkgb**  
#### But.. is there alternates ways?  
#  
> 🙋‍♂️ **fredkgb**  
#### Simple Scraper have an API  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/988804408093601864)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
